src/main/kotlin/fi/metatavu/timebank/api/keycloak/KeycloakController.kt

```diff
@@ -48,6 +50,44 @@ class KeycloakController {
         }
     }
 
+    /**
+     * Gets unspentVacations attribute for Person
+     * If not set will set and return default value of -1
+     *
+     * @param user UserRepresentation
+     * @return Int unspentVacations
+     */
+    fun getUsersUnspentVacationDays(user: UserRepresentation): Int {
+        return try {
+            user.attributes["unspentVacations"]!!.first()!!.toInt()
+        } catch (e: Exception){
+            updateUsersUnspentVacationDays(user, -1)
+            -1
```

Comment:
I believe @Jdallos initially requested these default values to be the ones calculated by the utilities provided by `VacationUtils`?